changelogs/fragments/1492_UserParameterDir _template.yml

```diff
@@ -0,0 +1,3 @@
+bugfix:
```

Comment:
sorry, truly don't mean to be a pain but the Sanity Tests will keep failing otherwise.  Correct category is `bugfixes`